apps/server/src/apps/helpers/app-logger-middleware.ts

```diff
@@ -0,0 +1,28 @@
+import { Request, Response, NextFunction } from 'express';
+import { Logger } from '@src/core/logger';
+import { RequestLoggable } from '@src/apps/helpers/request-loggable';
+import { Configuration } from '@hpi-schul-cloud/commons/lib';
+
+export const createAppLoggerMiddleware = (
+	logger: Logger
+): ((request: Request, response: Response, next: NextFunction) => void) => {
+	logger.setContext('AppLoggerMiddleware');
+	const enabled = Configuration.get('REQUEST_LOGGING_ENABLED') as boolean;
```

Comment:
Configuration.get ist das was wir aus den Code raus haben wollen. Was wäre der Vorschlag wenn es nicht mehr vorhanden ist? REQUEST_LOGGING_ENABLED gehört dann nicht mehr zu der Applikations Konfiguration.

Reply:
its too early to use any config framework that is not statically available at that point of initialization, you would have to go the path of doing it "proper" nestjs way, with all the bells and whistles, where this approach follows more the expressjs way to eliminate any performance cost before the request essentially hits nestjs proper